.github/workflows/build-and-deploy-pwa.yaml

```diff
@@ -0,0 +1,47 @@
+name: Build and deploy frontend
+
+on:
+  push:
+    branches:
+      - main
+
+jobs:
+  # Build Job
+  build:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v3
+      - name: Install Node
+        uses: actions/setup-node@v3
+        with:
+          node-version: 20
+      - name: Install Dependencies
+        run: cd power-pay-frontend && npm ci
+
+      - name: Build Project
+        run: cd power-pay-frontend && npm run build-for-gh
+
+      - name: Upload artifact to enable deployment
+        uses: actions/upload-artifact@v3
+        with:
+          name: production-files
+          path: power-pay-frontend
```

Comment:
Why taking this path? Why not `power-pay-frontend/build` ?

Reply:
I was aiming for a match between the path and the publish_dir. Should I also change the publish_dir to the same or just the path?